gen/injection.js

```diff
@@ -0,0 +1,166 @@
+            break;
+        case 1:
+            url = parse('/edge/v1/admin/service/:ServiceID/config/:ConfigSection', [ServiceID, ConfigSection]);
+            break;
+        case 2:
+            url = parse('/edge/v1/admin/service/:ServiceID/config/:ConfigSection/:ConfigKey', [ServiceID, ConfigSection, ConfigKey]);
+            break;
+        case 3:
+            url = parse('/edge/v1/admin/service/:ServiceID/config/:ConfigSection/:ConfigKey', [ServiceID,ConfigSection,ConfigKey]);
+            break;
+        default:
+            sys.logs.error('Invalid argument received.');
+            return;
+    }
+    sys.logs.debug('[veritone] GET from: ' + url);
+    var options = checkHttpOptions(url, httpOptions);
+    return endpoint._get(options);
+    //END_INJECTION
+}
+
+aiDataAdminOrganizationsOpenidRolesGet = () => {
+    //INIT_INJECTION
+    if (aguments.length === 0 || !id) {
+        sys.logs.error('Invalid argument received. This helper should receive the following parameters as non-empty strings: [id].');
+        return;
+    }
+
+    if(!httpOptions){
+        for (var i = 0 ; i < arguments.length; i++){
+            if (isObject(arguments[i])){
+                httpOptions = arguments[i];
+            }
+        }
+    }
+    var url;
+    switch(arguments.length-1){
+        case 0:
+            url = parse('/api/admin/organizations/:id/openid-roles', [id]);
+            break;
+        case 1:
+            url = parse('/api/admin/organizations/:id/openid-roles/:openidRoleKey', [id, openidRoleKey]);
+            break;
+        case 2:
+            url = parse('/api/admin/organizations/:id/openid-roles/:openidRoleKey', [id,openidRoleKey]);
+            break;
+        default:
+            sys.logs.error('Invalid argument received.');
+            return;
+    }
+    sys.logs.debug('[veritone] GET from: ' + url);
+    var options = checkHttpOptions(url, httpOptions);
+    return endpoint._get(options);
+    //END_INJECTION
+}
+
+voiceTtsVoiceGet = () => {
+    //INIT_INJECTION
+    if (aguments.length === 0 || !voiceId) {
+        sys.logs.error('Invalid argument received. This helper should receive the following parameters as non-empty strings: [httpOptions or voiceId].');
+        return;
+    }
+    if(!httpOptions){
+        for (var i = 0 ; i < arguments.length; i++){
+            if (isObject(arguments[i])){
+                httpOptions = arguments[i];
+            }
+        }
+    }
+    var url;
+    switch(arguments.length-1){
+        case 0:
+            url = parse('/v2/tts/voice?lang');
+            var options = checkHttpOptions(url, voiceId);
+            break;
+        case 1:
+            url = parse('/v2/tts/voice/:voice_id', [voiceId]);
+            var options = checkHttpOptions(url, httpOptions);
+            break;
+        case 2:
+            url = parse('/v2/tts/voice/:voice_id', [voiceId]);
+            var options = checkHttpOptions(url, httpOptions);
+            break;
```

Comment:
The variable 'options' should be declared outside the switch. And in the case of the 'voiceId' argument, shouldn't it be 'httpOptions'?

Reply:
I agree with the simplification of the options variable declaration.